content/en/device-provisioning-service/overview.md

```diff
@@ -0,0 +1,86 @@
+{{% note %}}
+Process described above varies from manufacturer to manufacturer. Nevertheless, the unique device's manufacturer identity is expected as the result. Additionaly, formulation "Creation of the device's manufacturer identity" was used to not confuse the reader if using more accurate term - "Provisioning of the device's manufacturer identity".
+{{% /note %}}
+DPS does not introduce a new step in the manufacturing process. It ties into the existing step creating the unique identifying key information - the device's manufacturer certificate or TPM's endorsment key used for device attestation during the zero-touch provisioning by the DPS.
+
+### Operation setup step
+
+[Creating an Enrollment Group]() in your plgd DPS instance is required in order to make sure the DPS can properly attest to the device's identity when it comes looking for its provisioning configuration. The operator is responsible to include the identifying key information described in the manufacturer step and add it to the enrollment group. Part of the enrollment group configuration is among others also initial resource and [ACL]() configuration, useful for the [device-to-device scenarios]() where the plgd hub is not involved at all. Once the enrollment group is configured, DPS is ready to automatically provision devices. Unless the use-case or list of devices changes, the DPS service / enrollment group does not have to be modified. Number of enrollment groups is not limited; different set of devices can be attestated by different enrollment groups containing different provisioning configuration.
+
+### Provisioning step
+
+The Enrollment Group created during the operation setup state has a set of required and optional configuration options, supporting various use-cases or security requirements. The following diagram describes what goes on behind the scenes to get a device securely provisioned. This flow is dependent on the Enrollment Group configuration, which is a prerequisite.
+
+{{< plantuml id="resource-publish" >}}
+@startuml
+autonumber
+skinparam backgroundColor transparent
+hide footbox
+
+actor "Solution Operator" as op
+collections "Devices" as d
+participant "Device Provisioning\nService" as dps
+participant "plgd hub" as hub
+
+note over op, hub
+Enrollment Entry including Devices key identifiers was created
+end note
+d -> dps: Connect and authenticate
+activate d
+dps -> dps: Find matching Enrollment Group
+activate dps
+d -> dps: Send Identity Certificate Signing Request
+d -> dps: Retrieve ACL configuration
+d -> d: Device ownership transfered
+d -> dps: Get access token for plgd hub API
```

Comment:
this is the last step -> we don't want to change resources during connecting to the cloud.